mkrelease.sh

```diff
@@ -82,21 +77,10 @@ chmod 444 $2
 }
 inst $tmpdir/${name}.tar.gz	$dst/${name}.tar.gz
 inst $tmpdir/${name}.tar.bz2	$dst/${name}.tar.bz2
```

Comment:
AFAICT, this whole script is very specific to Squid Project release process/environment. Consider moving the script itself from this repository into [squid-cache/ci](https://github.com/squid-cache/ci/tree/main/release), especially if release process improvements (i.e. script changes) may affect releases in multiple official Squid branches.

Reply:
The parts which are specific to the Squid Project release environment are being removed by this PR. What remains is to enable anyone to patch Squid sources and produce their own forked bundles for distribution.

Reply:
Assistance moving the logic from this script into Makefile.am dist rules is welcome as separate PRs.
The scope of this PR is simply to merge the changes we found necessary to actually release 6.0.2 and 5.9 this month so they do not cause problems again next release.